package.json

```diff
@@ -238,6 +238,7 @@
     "satisfied": "^1.1.1",
     "scheduler": "0.20.2",
     "schema-utils": "3.1.1",
+    "screener-storybook": "0.23.0",
```

Comment:
Why was this added?

Reply:
the commit describes it https://github.com/microsoft/fluentui/pull/24345/commits/a577dff9d11eebb7adb636194fe88e1f5713e960

- to align with our single version policy in monorepo. it's not related to scope of this PR but I did it anyway :D